manimpango/__init__.py

```diff
@@ -14,4 +15,9 @@
     from .enums import *  # noqa: F403,F401
     from .register_font import *  # noqa: F403,F401
 except ImportError:  # pragma: no cover
```

Comment:
we should also get the original error message and append to end of message

Reply:
Have a look at how numpy does: https://github.com/numpy/numpy/blob/cb557b79fa0ce467c881830f8e8e042c484ccfaa/numpy/core/__init__.py#L27

Reply:
> Have a look at how numpy does: https://github.com/numpy/numpy/blob/cb557b79fa0ce467c881830f8e8e042c484ccfaa/numpy/core/__init__.py#L27

Do you want it to be as verbose as numpy or just have similar elements?

Reply:
I would like it to be verbose as numpy.